src/dom.js

```diff
@@ -0,0 +1,257 @@
+const todoPriorityInput = document.querySelector('#task-priority');
+const todoNotesInput = document.querySelector('#task-notes');
+
+// Display Todos
+const displayTodosDiv = document.querySelector('#display-tasks');
+const todoContainer = document.querySelector('#todo-container');
+
+// Misc
+const areSure = document.querySelector('#are-sure');
+const yess = document.querySelector('#btn-yes');
+const noo = document.querySelector('#btn-no');
+
+const projectDelConfirm = document.querySelector('#project-del-confirm');
+const yesP = document.querySelector('#btn-p-yes');
+const noP = document.querySelector('#btn-p-no');
+
+// current project variable
+let currentProject = '';
+let todoTasks = [];
+let currentTodo = '';
+const toDltTodo = '';
+
+
+const showEditForm = (title) => {
+  currentTodo = '';
+  showForm(TodoFormDiv);
+  showForm(editForm);
+  hideForm(submitForm);
+  currentTodo = title;
+};
+
+// Empty the todos container, get the todos array from the storage,
+// loop through it and append a div with its content to the todos container
+const displayTodos = () => {
+  displayTodosDiv.innerHTML = '';
+
+  const btnDone = () => {
+    const one = document.createElement('button');
+    one.setAttribute('class', 'submit-btn');
+    one.addEventListener('click', () => disabledDiv(one));
+    one.innerHTML = 'Done/Undone';
+    return one;
+  };
+
+  const todos = JSON.parse(localStorage.getItem(currentProject));
+  todos.forEach((todo) => {
+    const todoDiv = document.createElement('div');
+    todoDiv.setAttribute('class', 'tasks-list');
+    todoDiv.setAttribute('id', 'tasks-list');
+
+    todoDiv.innerHTML = `
+      <h3 class="task-title" id="todo-title"><span class="todo-h3"> ${todo.title} </span></h3>
+      <div class="todo-details">
+        <p class="task-description"><span class="todo-titles"> Description:</span> ${todo.description}</p>
+        <p class="task-date"><span class="todo-titles">Date:</span> ${todo.date}</p>
+        <p class="task-date"><span class="todo-titles">Priority:</span> ${todo.priority}</p>
+        <p class="task-notes"><span class="todo-titles">Notes:</span> ${todo.notes}</p>
+      </div>
+      <button id="btn-edit-todo" class="submit-btn card-btn" data-name="${todo.title}">Edit</button>
+      <button id="btn-delete-todo" class="submit-btn btn-delete" data-delete="${todo.title}">Delete</button>
+      `;
+
+    displayTodosDiv.appendChild(todoDiv);
+    todoDiv.appendChild(btnDone());
+
+    const btnsEdit = document.querySelectorAll('#btn-edit-todo');
+    btnsEdit.forEach((btn) => {
+      btn.addEventListener('click', () => showEditForm(btn.dataset.name));
+    });
+
+    const btnsDelete = document.querySelectorAll('#btn-delete-todo');
+    btnsDelete.forEach((btn) => {
+      btn.addEventListener('click', () => showForm(areSure));
+    });
+  });
+};
+
+
+// Creating a new instance of the project then calling the local storage function
+const addProject = () => {
```

Comment:
Awesome job guys, you called this file `dom,js` which is great but you're having some logic inside, kindly separate :+1: 
Accordingly to Odin instructions; 
![image](https://user-images.githubusercontent.com/31897434/70099835-48922500-1630-11ea-8d75-9a6601b62793.png)